From: ProgrammerIn-wonderland <alice@alicesworld.tech>
Subject: [PATCH 8/8] debian.qcom-x1e: Enable CONFIG_SPI_HID module in annotations

Enable CONFIG_SPI_HID as module in Ubuntu packaging annotations for arm64.

Signed-off-by: ProgrammerIn-wonderland <alice@alicesworld.tech>
---
 debian.qcom-x1e/config/annotations | 1 +
 1 file changed, 1 insertion(+)
diff --git a/debian.qcom-x1e/config/annotations b/debian.qcom-x1e/config/annotations
index 6dc5c30..1a81423 100644
--- a/debian.qcom-x1e/config/annotations
+++ b/debian.qcom-x1e/config/annotations
@@ -11499,6 +11499,7 @@ CONFIG_SPI_FSL_LPSPI                            policy<{'arm64': 'm'}>
 CONFIG_SPI_FSL_QUADSPI                          policy<{'arm64': 'm'}>
 CONFIG_SPI_FSL_SPI                              policy<{'arm64': 'y'}>
 CONFIG_SPI_GPIO                                 policy<{'arm64': 'm'}>
+CONFIG_SPI_HID                                  policy<{'arm64': 'm'}>
 CONFIG_SPI_HISI_KUNPENG                         policy<{'arm64': 'm'}>
 CONFIG_SPI_HISI_SFC                             policy<{'arm64': 'm'}>
 CONFIG_SPI_HISI_SFC_V3XX                        policy<{'arm64': 'm'}>
-- 
2.45.0
